tests/share/models/factories.py

```diff
@@ -144,3 +144,38 @@ class TagFactory(ShareObjectFactory):
 
     class Meta:
         model = models.Tag
+
+
+class ThroughTagsFactory(ShareObjectFactory):
+    tag = factory.SubFactory(TagFactory)
+    creative_work = factory.SubFactory(AbstractCreativeWorkFactory)
+
+    class Meta:
+        model = models.ThroughTags
+
+
+class WorkIdentifierFactory(ShareObjectFactory):
```

Comment:
Are these duplicate factories purposely separate or meant to replace the ones in https://github.com/CenterForOpenScience/SHARE/blob/9aac202c7315442b97b58a275e9a7ea83f661e62/tests/share/normalize/factories.py?